helix-core/pom.xml

```diff
@@ -219,7 +219,6 @@
             <phase>package</phase>
             <goals>
               <goal>jar</goal>
-              <goal>test-jar</goal>
```

Comment:
What's the impact of removing test-jar here? Are we using this?

Reply:
There is no impact of removing the test-jar from this phase. 

As we only compiled classes are from the src directory only so we are good with just the `helix-core-xxx-jdk8.jar` and we dont care about `helix-core-xxx-jdk8-tests.jar`